src/renderer/views/Channel/Channel.js

```diff
@@ -50,20 +46,18 @@ export default defineComponent({
   name: 'Channel',
   components: {
     'ft-card': FtCard,
-    'ft-input': FtInput,
     'ft-select': FtSelect,
     'ft-flex-box': FtFlexBox,
     'ft-loader': FtLoader,
     'ft-element-list': FtElementList,
     'ft-age-restricted': FtAgeRestricted,
-    'ft-share-button': FtShareButton,
-    'ft-subscribe-button': FtSubscribeButton,
     'channel-about': ChannelAbout,
     'ft-auto-load-next-page-wrapper': FtAutoLoadNextPageWrapper,
+    ChannelDetails
```

Comment:
Is there a reason it's `ChannelDetails` and not `channel-details` like how we do every other component? Is there a plan to update other components to not use the dashes and just use the component name?

Reply:
I was mostly inspired by this section https://vuejs.org/guide/components/registration.html#component-name-casing. As for updating the other components, that's a vague plan I have but I thought we could do it at the same time we convert the respective components to the SFC setup style composition API, that way we can already tell from the file name what has been migrated. That's a long term plan though, there is no rush to convert everything straight away, I just thought it would be good to use the SFC style for that component as I was creating a new component anyway and it means we won't have to do it later on.

Reply:
Okay, sounds good, as long as we're planning to make it consistent eventually 🙂